source/conf.py

```diff
@@ -19,7 +19,9 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('/Users/fliao/Documents/Nidata/nidata/diffusion'))
```

Comment:
Your path is already absolute, so `abspath` won't do anything.
Try using `src_dir = os.path.dirname(__file__)` to get the directory of this file (`source`),
then use `os.path.join(src_dir, '..')` to get the root project directory..

Or, to get the `nidata` path, try importing `nidata`, then using `nidata.__file__`